package.json

```diff
@@ -91,9 +90,9 @@
     "eslint-plugin-react-hooks": "^1.6.0",
     "eslint-plugin-standard": "^4.0.0",
     "husky": "^1.0.1",
-    "jest": "^25.1.0",
+    "jest": "^26.0.1",
     "lint-staged": "^8.1.1",
-    "parcel-bundler": "^1.10.1",
+    "parcel-bundler": "^1.12.4",
```

Comment:
It turns out `yarn upgrade-interactive` doesn't do a great job of deduping.

Doing a `yarn remove` and then `yarn add` to re-install these dependencies (the only ones that had babel dependencies) properly dedupes the babel versions and makes the build work.

However, I cannot figure out why the websocket module's `require()` is still failing tests. Even going back to jest v25.1.0 and pushing everything else to be the same as master, it still fails now 🤷‍♂️. Perhaps also a good sign that we should not be creating a websocket connection in our tests :).